docs/docs/behaviors/sticky-key.md

```diff
@@ -28,11 +32,13 @@ You can use any keycode that works for `&kp` as parameter to `&sk`:
 
 ### Configuration
 
-You can configure a different `release-after-ms` in your keymap:
+You can configure a different `release-after-ms`, enable `quick-release` or enable `ignore-modifiers` in your keymap:
 
 ```
 &sk {
     release-after-ms = <2000>;
+    quick-release;
+    ignore-modifiers;
```

Comment:
I believe `ignore-modifiers;` are already set for `&sk` by default: https://github.com/zmkfirmware/zmk/blob/32c8737a2225ce03455d7df0145a96a451672c7b/app/dts/behaviors/sticky_key.dtsi#L15 
We should probably note this in the section where we mention the property.

For this example, we can change it to remove the property with  `/delete-property/` maybe?

Reply:
Can we think of any scenarios for which you'd want to disable `ignore-modifiers`? If not I'll just describe what it does and side-step why you'd want to do it.